server/src/main/java/org/elasticsearch/index/query/functionscore/DecayFunctionBuilder.java

```diff
@@ -210,6 +210,7 @@ private AbstractDistanceScoreFunction parseVariable(String fieldName, XContentPa
 
         // dates and time and geo need special handling
         parser.nextToken();
+        // TODO these ain't gonna work with runtime fields
```

Comment:
I think I forgot to run these down. I'll add it to my list. But I don't think it blocks merging. We may just never need these to be honest.

Reply:
I don't think it is worth the effort. I reviewed the leftover TODOs and I am fine with this one staying there.

Reply:
++